dist/tablesaw.js

```diff
@@ -389,10 +404,15 @@ if( Tablesaw.mustard ) {
 			if( priority && priority !== "persist" ) {
 				$cells.addClass( self.classes.priorityPrefix + priority );
 
-				$("<label><input type='checkbox' checked>" + $this.text() + "</label>" )
-					.appendTo( $menu )
-					.children( 0 )
-					.data( "tablesaw-header", this );
+				var $label = $( "<label>", {
+					text:  $this.text()
+				} );
+
+				$( "<input>", {
+					type: "checkbox"
+				} ).attr( "checked", "" ).prependTo( $label );
```

Comment:
Since jQuery 1.6 the [`$.prop()`](http://api.jquery.com/prop/#prop2) method is the proper way to set properties (checked, disabled, etc.) on elements.